templates/setenv.sh.erb

```diff
@@ -81,8 +81,13 @@ export JAVA_OPTS
 
 # Set the JVM arguments used to start Confluence. For a description of the options, see
 # http://www.oracle.com/technetwork/java/javase/tech/vmoptions-jsp-140102.html
+<%- if scope['confluence::java11'] %>
```

Comment:
the scope syntax is deprecated. You should be able to access the variable with:
```suggestion
<%- if @java11 %>
```

Please write two tests where $java11 is true/false and that checks which strings are written to the file

Reply:
I was keeping in sync with syntax on the rest of the module, but have updated my commit to newer style